distros/jazzy/overrides.nix

```
@@ -184,6 +192,51 @@ in {
     hash = "sha256-Qaz26F11VWxkQH8HfgVJLTHbHwmeByQu8ENkuyk5rPE=";
   };
 
+  moveit-core = rosSuper.moveit-core.overrideAttrs({
+    postPatch ? "", ...
+  }: {
+    postPatch = postPatch + ''
+      substituteInPlace CMakeLists.txt --replace-fail 'find_package(octomap 1.9.7...<1.10.0 REQUIRED)' 'find_package(octomap REQUIRED)'
+    '';
+  });
+
+  moveit-ros-occupancy-map-monitor = rosSuper.moveit-ros-occupancy-map-monitor.overrideAttrs({
+    postPatch ? "", ...
+  }: {
+    postPatch = postPatch + ''
+      substituteInPlace CMakeLists.txt --replace-fail 'find_package(octomap 1.9.7...<1.10.0 REQUIRED)' 'find_package(octomap REQUIRED)'
+    '';
+  });
+
+  osqp-vendor = lib.pipe rosSuper.osqp-vendor [
+    # Make CMakeLists.txt amenable to automatic patching by the next step.
+    (pkg: pkg.overrideAttrs ({ prePatch ? "", ... }: {
+      prePatch = prePatch + ''
+        sed -i 's/GIT_TAG.*/GIT_TAG v0.6.2/' CMakeLists.txt
```

What about being more explicit? Something like this:
```suggestion
        substituteInPlace CMakeLists.txt --replace-fail 'set(git_tag "v0.6.2")' 'set(git_tag "v0.6.2")' # fail when upstream version changes
        substituteInPlace CMakeLists.txt --replace-fail 'GIT_TAG ''${git_tag}' 'GIT_TAG v0.6.2'
```

---

Sounds good.